frontend/src/styles/hightlight-js-style.css

```diff
@@ -0,0 +1,73 @@
+.hljs {
+    display: block;
+    background: white;
+    padding: 0.5em;
+    color: #333333;
+    overflow-x: auto;
+}
+
+.hljs-comment,
+.hljs-meta {
+    color: #5d635a;
+}
+
+.hljs-variable,
+.hljs-template-variable,
+.hljs-strong,
+.hljs-emphasis,
```

Comment:
Why declare classes with no css?

Reply:
Actually I just realized this is applying css to multiple classes at once, but it seems like a lot compared to the previous one?

Reply:
Ok I am fine with this as it makes the css clearer.